concepts/Common/Integration Guides/70 Manage Azure Blob Storage with DevExtreme Components/20 Back-End Applicatoin (Basic Configuration).md

```diff
@@ -0,0 +1,47 @@
+
+All three examples in this article require a combination of a server-side application and a client-side appilcation.
+
+* The [devextreme-file-manager-azure-client-side-binding](https://github.com/DevExpress-Examples/devextreme-file-manager-azure-client-side-binding) repository stores the source code of the back-end application in the `Azure_Backend` directory.
+* The [devextreme-file-manager-azure-server-side-binding](https://github.com/DevExpress-Examples/devextreme-file-manager-azure-server-side-binding) repository stores the source code of the back-end application in the `Azure_Backend` directory.
+* The [devextreme-file-uploader-direct-upload-to-azure](https://github.com/DevExpress-Examples/devextreme-file-uploader-direct-upload-to-azure) repository stores the source code of the back-end application in the `AzureDirectUpload_Backend` directory.
+
+All three back-end applications in our repositories use the **.NET Framework** to communicate with Azure Blob Storage API. This article also uses .NET code examples to illustrate back-end development concepts.
```

Comment:
Just checking, but in the context of "Azure Blob Storage API", is "API" singular or plural?

Reply:
singular. I'll add the definite article, thanks for noticing!